sql/core/src/test/resources/sql-tests/inputs/pgSQL/case.sql

```diff
@@ -0,0 +1,267 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- CASE
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/case.sql
+-- Test the case statement
+--
+-- There are 2 join condition is missing in this test case. we set spark.sql.crossJoin.enabled=true.
```

Comment:
- `is missing` -> `which is missing` or `missed`?
- `we set` -> `We set`.

BTW, it's not clear about the relation between the missed one and `crossJoin`.

Reply:
How about rewriting it to the following text?
> This test suite contains two Cartesian products without using explicit CROSS JOIN syntax. Thus, we set spark.sql.crossJoin.enabled to true.